etl/steps/data/garden/survey/2024-04-01/dietary_choices_uk.meta.yml

```diff
@@ -0,0 +1,94 @@
+definitions:
```

Comment:
a general meta point - I've looked at the staging server for this dataset and you might want to consider adding DoDs for the definitions of dietary choices rather than writing out these long subtitles. I am guessing you are working on this with Hannah though so I'd ask her what she thinks. 
```
– Flexitarian: mainly vegetarian, but occasionally eat meat or fish.
– Pescetarian: eat fish but do not eat meat or poultry.
– Vegetarian: do not eat any meat, poultry, game, fish, or shellfish.
– Plant-based / Vegan: do not eat dairy products, eggs, or any other animal product.
```

Reply:
Good idea! I'll suggest it to Ed (he created the charts, I think).